src/app/api/collections/tree.py

```diff
@@ -84,9 +83,8 @@ def tree_from_elastic(node_id: uuid.UUID) -> list[CollectionNode]:
     response: Response = tree_search(node_id).execute()
 
     if response.success():
-        collection = map_elastic_response_to_model(
```

Comment:
I prefered this way, because if we remove glom, we only have to remove it in one place. Now, we have to do it in several places. Furthermore, I find "map_elastic_response_to_model" more telling, than the loop comprehension now chosen.

Reply:
just 5 lines above we already make heavy use of glom :p

Either way, I don't mind :)

Reply:
reverted back to use map_elastic_response_to_model